src/Reduced.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ElegantBro\Arrayee;
```

Comment:
I think we need sub-namespace, something like `ElegantBro\Arrayee\Aggregation`

Reply:
I will try fix it in next commit